packages/atlas/src/components/_crt/CrtPortfolioTable/CrtPortfolioTable.tsx

```diff
@@ -0,0 +1,168 @@
+import styled from '@emotion/styled'
+import { useMemo, useState } from 'react'
+
+import {
+  SvgActionBuyNow,
+  SvgActionMarket,
+  SvgActionMore,
+  SvgActionNotForSale,
+  SvgActionShoppingCart,
+  SvgActionTransfer,
+  SvgActionVerified,
+} from '@/assets/icons'
+import { Avatar } from '@/components/Avatar'
+import { FlexBox } from '@/components/FlexBox'
+import { NumberFormat } from '@/components/NumberFormat'
+import { Table, TableProps } from '@/components/Table'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { ContextMenu } from '@/components/_overlays/ContextMenu'
+
+const COLUMNS: TableProps['columns'] = [
+  { Header: 'Token', accessor: 'token', width: 150 },
+  { Header: 'Status', accessor: 'status', width: 200 },
+  { Header: 'Transferable', accessor: 'transferable', width: 100 },
+  { Header: 'Vested', accessor: 'vested', width: 100 },
+  { Header: 'Total', accessor: 'total', width: 100 },
+  { Header: '', accessor: 'utils', width: 70 },
+]
+
+export type PortfolioToken = {
+  tokenId: string
+  status: 'market' | 'sale' | 'idle'
+  vested: number
+  total: number
+  transferable: number
+}
+
+export type CrtPortfolioTableProps = {
+  data: PortfolioToken[]
+  isLoading: boolean
+}
+
+export const CrtPortfolioTable = ({ data }: CrtPortfolioTableProps) => {
+  const mappingData = useMemo(() => {
+    return data.map((row) => ({
```

Comment:
The `getTokenInfo` could be moved here because Transferable, Vested, and Total need to show the token title